src/main/java/com/palantir/gradle/gitversion/NativeGit.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.gitversion;
+
+import java.util.Map;
+
+interface NativeGit {
```

Comment:
If we only support native git, do we really need the "native" qualifier?

```suggestion
interface Git {
```

Reply:
@pkoenig10 Thanks for your review. However, currently we cannot merge this because there might be situations on highside where `git` binary is not installed. We cannot risk failure.